homeassistant/components/microbot_push/switch.py

```diff
@@ -0,0 +1,42 @@
+"""Switch platform for MicroBot."""
+from homeassistant.components.switch import SwitchEntity
+
+from .const import DEFAULT_NAME, DOMAIN, ICON
+from .entity import MicroBotEntity
+
+
+async def async_setup_entry(hass, entry, async_add_devices):
+    """Set up MicroBot based on a config entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_devices([MicroBotBinarySwitch(coordinator, entry)])
+
```

Comment:
```suggestion
async def async_setup_entry(hass, entry, async_add_entities):
    """Set up MicroBot based on a config entry."""
    coordinator = hass.data[DOMAIN][entry.entry_id]
    async_add_entities([MicroBotBinarySwitch(coordinator, entry)])

```

Please copy the typing from `switchbot/switch.py`